Tests/NIOTests/EventLoopTest.swift

```diff
@@ -156,23 +167,33 @@ public class EventLoopTest : XCTestCase {
         }
         let loop = group.next() as! SelectableEventLoop
 
-        let serverChannel = try ServerBootstrap(group: group).bind(host: "127.0.0.1", port: 0).wait()
+        let serverChannel = try ServerBootstrap(group: group)
+            .childChannelInitializer { channel in
+                channel.pipeline.add(handler: WedgeOpenHandler { promise in
+                    promiseQueue.sync { promises.append(promise) }
+                })
+            }
+            .bind(host: "127.0.0.1", port: 0).wait()
         defer {
             XCTAssertNoThrow(try serverChannel.syncCloseAcceptingAlreadyClosed())
         }
+        let connectPromise: EventLoopPromise<Void> = loop.newPromise()
 
         // We're going to create and register a channel, but not actually attempt to do anything with it.
-        let wedgeHandler = WedgeOpenHandler()
+        let wedgeHandler = WedgeOpenHandler { promise in promiseQueue.sync { promises.append(promise) } }
```

Comment:
nit: I like the formatting here more in terms of readability:

https://github.com/apple/swift-nio/pull/338/files#diff-6e227f3a42e326a8e6c1dc8acdd423e7R172